auction9-frontend/src/app/custom-validators/min-start-time-validator.ts

```diff
@@ -0,0 +1,19 @@
+import { FormGroup } from "@angular/forms";
+import * as moment from 'moment';
+
+export const minStartTimeValidator = (group: FormGroup): { [key: string]: boolean } => {
+  const startDate = group.get('dateGroup.startDate').value;
+  const startTime = group.get('startTime').value;
+
+  if (startTime != '') {
```

Comment:
I think that linter should report and require different usage of logical operator (3 signs, instead of 2). example: !==

Reply:
Fixed